### ISSUE TITLE
Keyboard Navigation skips the Schedule an appointment link 

### ISSUE THREAD

**Issue by ldelacosta** [department-of-veterans-affairs/va.gov-team#81077]
## Summary
When a user does not have any Upcoming, Pending or Past appointments, the frontend displays a message to the user. The keyboard navigation skips the `Schedule an appointment` link in the message. 

**Screenshot:** 
<img width="521" alt="Screenshot 2024-04-18 at 1 26 22 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/90797205/bb710e3e-77be-411f-8bd0-6423a8f2887a">

## Screen recording for reference: 
https://github.com/department-of-veterans-affairs/va.gov-team/assets/90797205/b1f1baf3-d9bb-454e-9e97-361887973be0

## Specs
- Device: Desktop
- Browser: Google Chrome

## Steps to Reproduce
Test user: [mhvmark.test@id.me](mailto:mhvmark.test@id.me)
Password: [File](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/sm/test-accounts.md)

1. Go to: https://staging.va.gov/my-health/appointments
2. Use the tab button  on the page

### Actual Result
When the user is navigating the page using the Tab key, it skips the `Schedule an appointment` link in the message. 

### Desired Result
The user should be able to navigate to the `Schedule an appointment` link in the content message. 

## Definition of Done
- [ ] Desired result is achieved and/or document decisions made
- [ ] Relevant documentation is updated

---

**Comment by JunTaoLuo** [comment]
Completed in https://github.com/department-of-veterans-affairs/vets-website/pull/30126.

**Comment by ldelacosta** [comment]
Re-opening the ticket to validate in staging. Thanks!

**Comment by ldelacosta** [comment]
Validated in staging:
<img width="1205" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNDgyQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b107c31df5dc0f5768e99223af5a9b3da0e5e94d/Screenshot%202024-06-04%20at%202.02.28%E2%80%AFPM.png" alt="Screenshot 2024-06-04 at 2.02.28 PM.png" />


No issues to report. Closing out
